qemu/tests/cfg/qsd_install.cfg

```diff
@@ -0,0 +1,55 @@
+- qsd_install:
+    type = qsd_install
+    virt_test_type = qemu
+    start_vm = no
+    only qcow2 raw
+    # QSD attributes
+    images = ""
+    qsd_namespaces = "qsd1"
+    qsd_images_qsd1 = "stg1"
+    qsd_force_create_qsd1 = yes
+    # Image attributes
+    image_name_stg1 = images/stg1
+    image_size_stg1 = 28G
+    vm_mem_backend = memory-backend-memfd
+    vm_mem_share = yes
+
+    # Image export attributes
+    qsd_image_export_stg1 = {"type":"vhost-user-blk","num-queues":1}
+    drive_format_stg1 = vhost-user-blk-pci
+    image_vubp_props_stg1=  {"num-queues":1}
+    # Install attributes
+
+    unattended_delivery_method = cdrom
+    medium = cdrom
+    cdroms += " unattended"
+    guest_cmd = lsblk
+    cd_format_fixed = ahci
+    drive_index_unattended = 1
+    drive_index_cd1 = 2
+
+    ovmf:
+        only q35
```

Comment:
Do we need to consider  aarch64 platform?

Reply:
So far, the main test is focused on x86, other platform is optional.
I will remove the limit in cfg.